spec/BatchRequestSpec.php

```diff
@@ -10,35 +10,35 @@
 
 class BatchRequestSpec extends ObjectBehavior
 {
-    function let(HttpPsrClient $client)
+    function let(HttpClient $client)
     {
         $this->beAnInstanceOf('spec\Http\Client\Utils\BatchRequestStub', [$client]);
     }
 
-    function it_send_multiple_request_using_send_request(HttpPsrClient $client, RequestInterface $request1, RequestInterface $request2, ResponseInterface $response1, ResponseInterface $response2)
+    function it_send_multiple_request_using_send_request(HttpClient $client, RequestInterface $request1, RequestInterface $request2, ResponseInterface $response1, ResponseInterface $response2)
     {
-        $client->sendRequest($request1, [])->willReturn($response1);
-        $client->sendRequest($request2, [])->willReturn($response2);
+        $client->sendRequest($request1)->willReturn($response1);
+        $client->sendRequest($request2)->willReturn($response2);
 
-        $this->sendRequests([$request1, $request2], [])->shouldReturnAnInstanceOf('\Http\Client\BatchResult');
+        $this->sendRequests([$request1, $request2])->shouldReturnAnInstanceOf('\Http\Client\BatchResult');
```

Comment:
Slash is not required in case of strings: they are imported fromthe root namespace anyway.